modules/phase_field/include/kernels/CHInterfaceBase.h

```diff
@@ -67,9 +67,9 @@ CHInterfaceBase<T>::CHInterfaceBase(const InputParameters & parameters)
     _M(getMaterialProperty<T>("mob_name")),
     _dMdc(getMaterialPropertyDerivative<T>("mob_name", _var.name())),
     _d2Mdc2(getMaterialPropertyDerivative<T>("mob_name", _var.name(), _var.name())),
-    _second_u(second()),
-    _second_test(secondTest()),
-    _second_phi(secondPhi()),
+    _second_u(_var.secondSln()),
+    _second_test(_var.secondPhi()),
+    _second_phi(_var.secondPhi()),
```

Comment:
And there is another reason why `MooseVariableInterface` should have stayed...

Reply:
But it's only 5 more chracters vs. a whole intermediate class? Now the repeated call of `_sys.getVariable(_tid, parameters.get<NonlinearVariableName>("variable"))` (or `_subproblem...`) in all the computing object classes does seem tedious I'll agree

Reply:
And let's not forget we might hide some smart things in that `MooseVariableInterface` c-tor. Those would get propagated into all interfaces that are using the class...